chsdi/models/vector/__init__.py

```diff
@@ -21,14 +21,13 @@
 Geometry2D = GeometryChsdi(geometry_type='GEOMETRY', dimension=2, srid=2056)
 Geometry3D = GeometryChsdi(geometry_type='GEOMETRYZ', dimension=3, srid=2056)
 
-
 MAX_FEATURE_GEOMETRY_SIZE = 1e6
 
 DEFAULT_OUPUT_SRID = 21781
 
 
 def get_resolution(imageDisplay, mapExtent):
-    # TODO: Python2/3
+    # TODO: clean-up when only Python 3.x and no longer 2.x is in use
     if not isinstance(imageDisplay, list):
         imageDisplay = list(imageDisplay)
```

Comment:
Here also the comment and `if not isinstance()` are useless, analyzing the code show that `imageDisplay` is always a list.

Reply:
Not sure about that. For example in the `cadastralwebmap_opendata.mako` the default value for `imageDisplay` is a string ('400,600,96'). Hence I'd suggest to leave the `ìsinstance()` here.

Reply:
No in `cadastralwebmap_opendata.mako` it is a list as well, you have a `setter` for `imageDisplay` that parse and translate strings into list, see https://github.com/geoadmin/mf-chsdi3/blob/ff57e114f3827716c280c990e1fcbdab77dac646/chsdi/lib/validation/identify.py#L175 and https://github.com/geoadmin/mf-chsdi3/blob/ff57e114f3827716c280c990e1fcbdab77dac646/chsdi/lib/validation/identify.py#L79
IMHO this should be removed now has this is not needed for python2 and `isinstance` is not cheap.

Reply:
done